graphql-server/migrations/1588378316564_update_permission_anonymous_public_table_players/up.yaml

```diff
@@ -0,0 +1,23 @@
+- args:
+    role: anonymous
+    table:
+      name: players
+      schema: public
+  type: drop_select_permission
+- args:
+    permission:
+      allow_aggregations: false
+      columns:
+      - client_uuid
+      - game_id
+      - id
+      - team
+      - team_sequence
+      - username
```

Comment:
@avimoondra I was getting an error when fetching the current player, since it looks like the anonymous user doesn't have access to the `username` or `team_sequence` field.

That error happens currently if you try to go to a game lobby when don't have a user stored in localStorage:
<img width="795" alt="Play_the_Fishbowl_Game__aka_Salad_Bowl__-_Online___Free" src="https://user-images.githubusercontent.com/3579673/80854430-1a1f7280-8bed-11ea-9288-3c7d267c57ea.png">

It looks like, when we were using the non-lazy version of this query (`const { data, loading } = useCurrentPlayerQuery({`),  the way to error handle was just to also unpack the `error` object (`const { data, loading, error } = ...`).

With the lazy version, you need to pass an `onError` callback, which means when there _is_ an error, the [`onComplete` callback](https://github.com/avimoondra/fishbowl/pull/22/files#diff-df1ab9a06b4534c03fc258438684a7afR52) doesn't fire. 

So tl;dr I'm pretty sure we need to fix these permissions issues so that we can properly handle fetching the current user when there is none.